modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/buffer/BufferedIndexerInvocationHandler.java

```diff
@@ -210,6 +210,12 @@ private void _bufferRequest(
 			IndexerRequestBuffer indexerRequestBuffer)
 		throws Exception {
 
+		float minimumBufferAvailabilityPercentage = 0.90F;
```

Comment:
This change doesn't make sense. There is a set method to that property, why did you do this change?